CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### App Center
 
 * **[Fix]** Handle incorrect usage of `AppCenter.setLogUrl` API to provide readable error message.
+* **[Fix]** Check for running in App Center Test will now work on newer Android SDKs.
```

Comment:
We already prepared (and thus froze code) for release 2.1.0, so this will have to be moved to 2.2.0.